model/qti/ParserFactory.php

```diff
@@ -1385,6 +1410,15 @@ private function buildImg(DOMElement $data){
         return $returnValue;
     }
 
+    private function buildTable(DOMElement $data){
+
+        $attributes = $this->extractAttributes($data);
+        $table = new Table($attributes);
+        $this->parseContainerStatic($data, $table->getBody());
```

Comment:
Tables are parsed in 2 differents places : first we only parse tables that contains interactions, then we parse the other tables (see previous comment)